.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -12,13 +12,29 @@ assignees: ''
 **Is your feature request related to a problem? Please describe.**
 A clear and concise description of what the problem is.
 
+**How it works Now:** Show the current checkstyle running on your code,
+so we could know more about your feature request and how checkstyle is
+behaving currently.
+
+[Download and run cli]('https://checkstyle.org/cmdline.html#Download_and_Run')
+
+```bash
```

Comment:
I would make it:
```
**How it works Now:** 
````bash
```

Reply:
The ci/cd pipeline is not letting it to do. It is creating markdown linting errors.
code should be surrounded with new line.